### PR TITLE
chore: create ES module build

### PR DESCRIPTION
**Pull request by mallachari**
Fixes: https://github.com/stoplightio/elements/issues/1392

Builds ES module along with CJS using sl-scripts bundling. 
Adds `styles.min.css` to exports to include it in the encapsulated bundle.

Tested with examples and platform (requires replacing deep imports for elements-core).

### CODE REVIEW

File: packages/elements-core/src/hooks/useComputeMarkdownHeadings.ts [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
   return React.useMemo(() => computeMarkdownHeadings(tree), [tree]);
 }
 
 export function computeMarkdownHeadings(tree: MDAST.Root): IArticleHeading[] {
-  return selectAll(':root > [type=heading]', tree)
+  return (selectAll(':root > [type=heading]', tree) as MDAST.Heading[])
     .map((heading: MDAST.Heading) => ({
       title: findTitle(heading),
-      id: heading.data && (heading.data.id as string | undefined),
+      id: (heading.data?.id as string | undefined) || '',
```
Comment (marcelltoth) [inline packages/elements-core/src/hooks/useComputeMarkdownHeadings.ts:16]: What is this change? Is this related?

File: packages/elements/package.json [hunk 2]
```diff
@@ -96,5 +96,8 @@
   },
   "release": {
     "extends": "@stoplight/scripts/release"
+  },
+  "exports": {
```
Comment (marcelltoth) [inline packages/elements/package.json:100]: Aren't you missing the equivalent for `dev-portal`?

**Review by marcelltoth — COMMENTED** [review]
Looks great, 2 blockers, and I'd have to try installing it somewhere. Have you?

File: packages/elements-core/src/hooks/useComputeMarkdownHeadings.ts (hunk 1 above)
Comment (mallachari) [reply packages/elements-core/src/hooks/useComputeMarkdownHeadings.ts:16]: I've replaced `unist-util-select`, earlier imported by `require` and having no typing. This change was to have some typing instead using `any`

File: packages/elements/package.json (hunk 2 above)
Comment (mallachari) [reply packages/elements/package.json:100]: Yeah, right. We also export styles in dev-portal. I'll add it

**Comment by mallachari** [comment]
I've installed it in examples and platform. In platform we're using deep imports so it has to be updated there (but all components are accessible directly from index file)

**Comment by marcelltoth** [comment]
> I've installed it in examples and platform. In platform we're using deep imports so it has to be updated there (but all components are accessible directly from index file)

Thank you! I marked your issue to be blocked by stoplightio/platform-internal#7072 It's in our sprint anyway, so you might as well just commit your test code and complete two issues with the same effort ;)

**Comment by mallachari** [comment]
I forgot about that issue. Yup, I'll do that too

**Comment by mallachari** [comment]
Unblocked, ready to go.

**Comment by marcelltoth** [comment]
Doing one more check, then ✅

**Review by marcelltoth — APPROVED** [review]
Looks good!

**Comment by mallachari** [comment]
Should we also bump the packages within this PR?

**Comment by marcelltoth** [comment]
Let's do that separately, remember last time:)
